packages/actor-init-sparql/config/sets/rdf-source-identifiers.json

```diff
@@ -18,6 +18,17 @@
         "@id": "config-sets:http.json#mediatorHttp"
       }
     },
+    {
+      "@id": "config-sets:rdf-source-identifiers.json#myActorRdfSourceIdentifierHypermediaNextPage",
+      "@type": "ActorRdfSourceIdentifierHypermediaQpf",
+      "carsihq:Actor/RdfSourceIdentifier/HypermediaQpf/mediatorHttp": {
+        "@id": "config-sets:http.json#mediatorHttp"
+      },
+      "carsihq:Actor/RdfSourceIdentifier/HypermediaQpf/toContain": [
+            "http://www.w3.org/ns/hydra/core#",
+            "next"
+      ]
+    },
```

Comment:
Until https://github.com/comunica/comunica/issues/121#issuecomment-394976570 is fixed, this is indeed the only way to achieve the desired behaviour.

However, this will unfortunately require an additional HTTP request _for each query_ we do, which is not really desired.
We want to avoid bloating the default Comunica engine too much.

I would suggest not adding this next-page actor (and source identifier) to the default config, and instead create a dedicated (external) config for this. (For example: https://github.com/RubenVerborgh/LDflex-Comunica/commit/14624cfaf77402ee813533bcf566abb974d676c5)

@pietercolpaert Would creating a custom Comunica instance somewhere based on a separate config file make sense for your use case?

Reply:
And also a separate one then for every new hydra:search form or hypermedia case we come up with? I think that would be detrimental to the idea of Comunica, although it could be a temporary fix.

Is it another option to prioritize https://github.com/comunica/comunica/issues/121#issuecomment-394976570 and fix this this week?

Reply:
> Is it another option to prioritize #121 (comment) and fix this this week?

Not on my end no. But @Lamasaurus is welcome to have a go at it.